grails-app/controllers/streama/FileController.groovy

```diff
@@ -84,6 +84,17 @@ class FileController {
     respond status: NO_CONTENT
   }
 
+  def removeMultipleFilesFromDisk() {
+    def bulk = params.files
+    def listOfFiles = JSON.parse(bulk)
+
+    listOfFiles.each { element ->
```

Comment:
and then here it would be the ids that you iterate, which makes things easier to name. instead of "element" you can call it "id" etc.